core/src/tile/tileWorker.cpp

```diff
@@ -94,10 +97,10 @@ void TileWorker::run() {
 
             // float loadTime = (float(clock() - begin) / CLOCKS_PER_SEC) * 1000;
             // LOG("loadTime %s - %f", task->tile->getID().toString().c_str(), loadTime);
-
-            m_tileManager.tileProcessed(std::move(task));
```

Comment:
And `requestRender` should be called when tile is finished building, and hence moved inside the scope of the above if condition.